### PR TITLE
Localisation documentation alternative

### PR DESCRIPTION
**Pull request by peteruithoven**
I had trouble figuring this out in a new project without the pot and po files. Seemed like there was a build in method to generate those. I understand that I might have made a few controversial choices like using a `LINGUAS` file and creating a sub `meson.build` file. I also don't fully understand why the `GETTEXT_PACKAGE` global wasn't mentioned.

### CODE REVIEW

**Comment by jpakkane** [comment]
@TingPing does this seem ok to you?

File: docs/markdown/Localisation.md [hunk 1]
```diff
@@ -4,37 +4,57 @@ short-description: Localization with GNU Gettext
...
 
-We also need to define an array of strings containing all the locales we want to generate. This is done in the Meson file in the `po` subdirectory. Assuming we want to generate Finnish and German localisations, the definition would look like this.
-
+### meson.build
+Localization specific meson file. It imports and uses the `i18n` module. If not defined before it needs to define the `GETTEXT_PACKAGE` global.
 ```meson
-langs = ['fi', 'de']
+i18n = import('i18n')
+# define GETTEXT_PACKAGE global
+add_global_arguments('-DGETTEXT_PACKAGE="intltest"', language:'c')
```
Comment (jpakkane) [inline docs/markdown/Localisation.md]: This should be `add_project_arguments` instead.

Comment (peteruithoven) [reply docs/markdown/Localisation.md]: Fixed :) I was expecting way more issues.

**Comment by peteruithoven** [comment]
Hey guys, could you check this again?
I'm not sure why the travis build failed, since this is docs only.

**Comment by jpakkane** [comment]
Seems ok to me but it would be nice to get an ack from someone with domain experience.

**Comment by jpakkane** [comment]
Thanks.

**Pull request merged**